tickets/src/utils/dates.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * This function returns the future days, months and up to 3 years based on a current
+ * date.
+ * @param {*} now
+ * @param {*} year
+ * @param {*} month
+ */
+export const getDaysMonthsAndYears = (now, year, month) => {
+  let monthNames = [
+    'Jan',
+    'Feb',
+    'Mar',
+    'Apr',
+    'May',
+    'Jun',
+    'Jul',
+    'Aug',
+    'Sep',
+    'Oct',
+    'Nov',
+    'Dec',
+  ]
+
+  // Years are the next 3 years
+  const years = [
+    now.getFullYear(),
+    now.getFullYear() + 1,
+    now.getFullYear() + 2,
+  ].map(d => d.toString())
+
+  let months = [...monthNames]
+
+  if (year === `${now.getFullYear()}`) {
```

Comment:
This might be more explicit that we are considering the first year of our range?
```suggestion
  if (year === years[0]) {
```